src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/PrefetchGetRecordsCache.java

```diff
@@ -20,61 +20,68 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.LinkedBlockingQueue;
 
+import com.amazonaws.SdkClientException;
 import com.amazonaws.services.kinesis.clientlibrary.types.ProcessRecordsInput;
 import com.amazonaws.services.kinesis.model.GetRecordsResult;
 
 import lombok.NonNull;
 import lombok.extern.apachecommons.CommonsLog;
 
 /**
- * This is the default caching class, this class spins up a thread if prefetching is enabled. That thread fetches the
- * next set of records and stores it in the cache. The size of the cache is limited by setting maxSize i.e. the maximum
- * number of GetRecordsResult that the cache can store, maxByteSize i.e. the byte size of the records stored in the
- * cache and maxRecordsCount i.e. the max number of records that should be present in the cache across multiple
- * GetRecordsResult object. If no data is available in the cache, the call from the record processor is blocked till
- * records are retrieved from Kinesis.
+ * This is the prefetch caching class, this class spins up a thread if prefetching is enabled. That thread fetches the
+ * next set of records and stores it in the cache. The size of the cache is limited by setting
+ * maxPendingProcessRecordsInput i.e. the maximum number of GetRecordsResult that the cache can store, maxByteSize
```

Comment:
Remember that JavaDoc is HTML, so you can use HTML elements.  I would recommend describing the configuration like
``` html
<dl>
  <dt>maxPendingProcessRecordsInput</dt>
  <dd>The maximum number of ProcessRecordsInputs that can be in the cache at one time</dd>
  <dt>maxByteSize</dt>
  <dd>...</dd>
</dl>
```